src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -6097,6 +6097,7 @@ def test_aks_create_with_pod_identity_enabled(
         # create
         create_cmd = (
             "aks create --resource-group={resource_group} --name={name} --location={location} "
+            "--network-plugin kubenet "
```

Comment:
Should we leave this blank and let defaulting do its thing?

Reply:
Are we asserting the result of this create has kubenet?

Reply:
the reason I added kubenet explicity is because we're asserting this `self.check("podIdentityProfile.allowNetworkPluginKubenet", True)` so it seems like we have some property around pod identity specific to kubenet. I am not sure if that property is set if you had overlay (my guess is no)